index.src.html

```diff
@@ -289,6 +289,17 @@ <h4 id="concept-reports">Reports</h4>
   counter, which is a non-negative integer representing the number of times the
   user agent attempted to deliver the report.
 
+  <h4 id="concept-configuration">User configuration</h4>
+
+  Reporting may be <dfn for="reporting">disabled</dfn> for particular [=report
+  types=], potentially on particular [=origins=]. The specification does not
+  define how [=/user agents=] determine whether their users prefer to disable
+  reporting.
```

Comment:
This is the bit that I think addresses #168. While it doesn't go as far as @pes10k wants in requiring all UAs to make reporting opt-in, it does allow any UA to do that if it thinks that's what its users want. I think that's likely to be as far as the WG will have consensus to go, although the WG could surprise me.